CHANGELOG.md

```diff
@@ -20,8 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * [Polymer] Extract pseudo elements from HTML comments
 
 ### Fixed
-* Fixed a class of race conditions and cache invalidation errors that can occur when there are concurrent analysis runs and edits to files.
-
+* Fixed a class of race conditions and cache invalidation errors that can occur when there are parallel analysis runs and edits to files.
```

Comment:
pedantic: I think "concurrent" is the right term here. _If_ anyone intends there to be a difference, it's usually that concurrent operations are not necessarily parallel, and parallel actually means truly at the same time.